code/modules/clothing/suits/toggles.dm

```diff
@@ -4,6 +4,7 @@
 	actions_types = list(/datum/action/item_action/toggle_hood)
 	var/obj/item/clothing/head/hooded/hood
 	var/hoodtype = /obj/item/clothing/head/hooded/winterhood //so the chaplain hoodie or other hoodies can override this
+	var/toggle_slot = ITEM_SLOT_OCLOTHING
```

Comment:
This does not meet our [documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md).

Reply:
doc yer code